static/app/views/onboarding/onboarding.tsx

```diff
@@ -251,6 +251,18 @@ function Onboarding(props: Props) {
         ...onboardingContext.data,
         projects: currentProjects,
       });
+      if (docsOnPlatformClickEnabled) {
+        onboardingContext.setData({
+          ...onboardingContext.data,
+          projects: currentProjects,
+          selectedSDK: undefined,
```

Comment:
this also fixes the platform being selected when going back to the 'platform select' page. The deletion request shouldn't fail and the [issue](https://github.com/getsentry/projects/issues/813) was created to investigate the problem.